assets/prototype/domain/eventEditor/datetimes/DatesList/filterBar/controls/DatesSortedByControl.tsx

```diff
@@ -1,18 +1,26 @@
+import React, { useMemo } from 'react';
```

Comment:
This is not an external import?

Reply:
It should be below, under:
```
/**
 * External imports
 */
```

Reply:
Yes, it is

Reply:
AFAIC we don't need to add those comment blocks in to identify External vs Internal imports.

I **DO** think we should separate them though with a single line space and put them in alphabetical order based on the name of the lib.

Those comment blocks like Vadim posted above were something Darren wanted because that's what WP core did in the GB code.

Reply:
I am ok without those comments, I thought it is some kind of unwritten convention on the project.